packages/strapi-plugin-content-type-builder/controllers/validation/relations.js

```diff
@@ -5,10 +5,25 @@ const { validators, isValidName } = require('./common');
 const { typeKinds } = require('./constants');
 
 const REVERSE_RELATIONS = ['oneToOne', 'oneToMany', 'manyToOne', 'manyToMany'];
+const STRAPI_USER_RELATIONS = ['oneWay', 'manyWay'];
```

Comment:
Can you describe this a bit more? We are locking down the relations that can be used with the Admin panel users model?

Reply:
Yes :) For the moment there are no possiblities for the user to make a relation between an admin and another content-type. The idea is to let them do it without modifying the admon model (we want to keep the control of this model) which can be done only with oneWay and manyWay